File: Practice/AllDomains/Tutorials/30DaysOfCode/Day16ExceptionsStringtoInteger.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
class Solution 
{
	/*	author:@shivkrthakur  */
    static void Main(String[] args) 
	{
        string inputString = Console.ReadLine();
        try
            {
                int x = Convert.ToInt32(inputString);
            Console.WriteLine(x);
        }
        catch
            {
            Console.WriteLine("Bad String");
        }
    }
}

File: Practice/AllDomains/Tutorials/30DaysOfCode/Day16ExceptionsStringtoInteger.js
/*	author:@shivkrthakur  */
process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = "";
var input_stdin_array = "";
var input_currentline = 0;

process.stdin.on('data', function (data) {
    input_stdin += data;
});

process.stdin.on('end', function () {
    input_stdin_array = input_stdin.split("\n");
    main();
});

function readLine() {
    return input_stdin_array[input_currentline++];
}

/////////////// ignore above this line ////////////////////

function main() {
    /*
     * Try using `eval`
     */
    var S = readLine();

    try{
        console.log(eval(S));
    }
    catch(e){
        console.log("Bad String");
    }
}
